### ISSUE TITLE
MSE Loss is not capable of handling ignored targets

### ISSUE THREAD

**Issue by SebieF** [sacdallago/biotrainer#21]
The PyTorch nn.MSELoss is not able to handle ignored indices (like the nn.CrossEntropyLoss). 

A solution provided by @mheinzinger would be as follows:
>You could easily implement this by yourself by passing "reduce=None" when initializing your loss function. As a result it will return a loss for each residue in your input instead of a single scalar. Then you can manually multiply the masked elements by 0 and average. When averaging, divide only by the number of non-masked residues. 

This must be kept in mind for implementing residue(s)_to_value protocols in the future.

**Comment by SebieF** [comment]
Fixed in v1.1.0 via adding a manual masked_mse_loss: https://github.com/sacdallago/biotrainer/blob/v1.1.0/biotrainer/losses/masked_mse_loss.py
